openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -28,13 +28,13 @@ class CreateReview(plugin.Creator):
 
     def __init__(self, *args, **kwargs):
         super(CreateReview, self).__init__(*args, **kwargs)
+        data = OrderedDict(**self.data)
 
         # get basic animation data : start / end / handles / steps
-        data = OrderedDict(**self.data)
-        animation_data = lib.collect_animation_data(fps=True)
-        for key, value in animation_data.items():
+        for key, value in lib.get_frame_range().items():
```

Comment:
1. Could we add a comment here as to why we're using `get_frame_range` over `collect_animation_data`.
It's solely because we don't want to default it based on the timeline, but have it default to the asset?

2. Could we actually add this behind a Creator setting? Because our studio would prefer it to still be based on the current user timeline.

Reply:
Yeah, good point. Have done backwards compatibility now. New behaviour is behind a setting.

I was wondering though whether the timeline below, should result in a start handle of `25`?

![Capture](https://user-images.githubusercontent.com/1860085/221786961-f2df2068-cc89-4b28-a65e-f81d164d3bc6.PNG)

Reply:
I think so, yes. 1026-1001.

Reply:
It does not currently account for the Maya "handles" so we should have that behind a setting as well?

Reply:
> What does it do now, take the longest one? I think we've used it forever like that so that animators can focus on a part of the timeline but not have that interfere with what they are about to create to publish.

Yeah, it takes the longest timeline so its always without handles.

> To me the behavior where they don't turn into handles is just fine. What does the rest of OP crew think?

@antirotor @mkolar ?

Reply:
> Good question. What does it do now, take the longest one? I think we've used it forever like that so that animators can focus on a part of the timeline but not have that interfere with what they are about to create to publish.

This is only on instance creation though. So the animator can have whatever timeline when the instances are setup.